src/main/java/org/opentripplanner/raptor/api/request/MultiCriteriaRequest.java

```diff
@@ -20,16 +21,15 @@ public class MultiCriteriaRequest<T extends RaptorTripSchedule> {
   @Nullable
   private final RaptorTransitPriorityGroupCalculator transitPriorityCalculator;
 
-  @Nullable
-  private final PassThroughPoints passThroughPoints;
+  private final List<PassThroughPoint> passThroughPoints;
```

Comment:
I'm a little concerned seeing `PassThroughPoints' go: we're going from OO encapsulation to a 'primitive' list. If in the future we would like to attach more information or functionality, it would require substantial refactorization.

Reply:
Agree, and that is way we did it. But, Raptor does not need any more information so we should not pass this information into Raptor. So, the design is not good. When we introduce more info, then is the time to refactor this back. And I have added the `PassThroughPoints`in the optimized transfer service instead, but that is part of the next PR.